docs/deploying.md

```diff
@@ -481,6 +481,25 @@ OpenStackDataPlaneNodeSet has the following status fields:
 | DNSClusterAddresses | |
 | CtlplaneSearchDomain | |
 
+##### OpenStackDataPlaneNodeSet Pods
+
+Pods within OpenStackDataPlaneNodeSet can fail for many reasons.
+Under normal circumstances, these failures may be transient.
+However, for facilitating triage, the operator does check on service pods and reports
+if their status is indicative of failure.
```

Comment:
By service pod, do you mean the pod started by the job by the AnsibleEE by the Deployment for a DataPlaneService execution? This isn't clear.

Reply:
Right, the multiple levels of delegation are hard to keep track of, I better define it at the start.